src/components/ContestTile.js

```diff
@@ -55,27 +54,30 @@ const ContestTile = ({ contest: { node }, updateContestStatus }) => {
           />
         ) : (
           <p>
-            Contest ran {t.startDay}-{t.endDay}
+            Contest ran {t.startDay === "" ? "" : t.startDay}-
```

Comment:
It might make more sense to be more explicit about the.. state semantics? not (soon || active) might be unintentionally broad. It seems like it used to mean done, so a start/end day was guaranteed.